src/main/java/racingcar/collection/RaceResult.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.collection;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RaceResult {
+    private final List<Winner> winners = new ArrayList<>();
+    private final List<LapResult> labResults = new ArrayList<>();
+
+    public void addRaceResult(LapResult result) {
+        labResults.add(result);
+    }
+
+    public List<LapResult> getLabResults() {
+        return labResults;
```

Comment:
그대로 리스트를 반환하면 위변조에 취약합니다. 
```suggestion
        return new ArrayList<>(Arrays.asList(labResults)); //추가 삭제가 가능한 방식
        return Arrays.asList(labResults)); //추가 삭제가 불가능한 방식
```